src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -226,7 +259,7 @@ static void uct_rdmacm_cm_handle_event_connect_response(struct rdma_cm_event *ev
         ucs_error("rdma_establish on ep %p (to server addr=%s) failed: %m",
                   cep, ucs_sockaddr_str(remote_addr, ip_port_str,
                                         UCS_SOCKADDR_STRING_LEN));
-        uct_rdmacm_cm_ep_client_connect_cb(cep, &remote_data, UCS_ERR_IO_ERROR);
+        cep->disconnect_cb(&cep->super.super, cep->user_data);
```

Comment:
pls add code comment why we're calling disconnect: because connect_cb was already called